src/app/app.controller.ts

```diff
@@ -23,4 +28,12 @@ export class AppController {
     healthz(): string {
         return HttpConstants.HEALTHZ_RESPONSE;
     }
+
+    /**
+     * For the uptime statists report (see Uptime feature brief)
```

Comment:
No one outside of Kiva will have a view into the `Uptime feature brief` so I don't think that belongs in the comment here.

Also, small typo: "statistics" instead of "statists"